### PR TITLE
Tree homework

### PR DESCRIPTION
**Pull request by Annika-Klu**
Hi Ugur,
sorry I am adding this just now. First I didn't find the time to finish it, and eventually forgot about it. Since I failed to take notes on the requirements of this homework, I am not sure whether I did everything that was requested - if there is an aspect missing pls let me know :)
Thanks again for the class! It was so much fun and I feel I learned a lot.
Cheers
Annika

Screenshot 2020-10-04 at 16 17 18

### CODE REVIEW

**Comment by ugurtekbas** [comment]
Hey @Annika-Klu ,

Thank you very much for your PR. Actually you're the only one who did this homework and created the PR.
Code looks pretty neat and it works as expected. Kudos to you 🎉 🎉 
Before I merge this PR, there is a tiny issue in html file, name of the css file is not `tree.css` but `annika-tree.css`. You'll need to change that.
I will add a screenshot to description so we can see how it looks.

You're really good at programming, I'm sure you'll accomplish what you want in this field.

Cheers,
Ugur

File: annika-tree.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="tree.css" type="text/css">
```
Comment (ugurtekbas) [inline annika-tree.html]: This should be `annika-tree.css`

**Review by ugurtekbas — CHANGES_REQUESTED** [review]

File: annika-tree.html (hunk 1 above)
Comment (Annika-Klu) [reply annika-tree.html]: Okay, I've changed it - thank you very much! :)

**Review by ugurtekbas — APPROVED** [review]
Thanks a lot for doing this homework.

**Pull request merged**